scripts/auto_translation_skills.py

```diff
@@ -0,0 +1,181 @@
+import os
+import shutil
+from os.path import join, exists, dirname
+from pathlib import Path
+import re
+from ovos_utils.bracket_expansion import expand_template
+from ovos_translate_plugin_deepl import DeepLTranslator
+
+"""
+Usage:
+
+export TARGET_LANG=uk-ua
+export SKILL_DIRECTORY=/home/$USER/skill-directory
+python3 translate.py
+"""
+
+API_KEY = '6f0f043a-6274-4527-b8fc-95720b03f70c'
+if not API_KEY:
+    raise ValueError("API Key is not specified")
+
+tx = DeepLTranslator({"api_key": API_KEY})
+
+single_lang = os.getenv("TARGET_LANG")
+target_langs = (single_lang,) if single_lang else (
+    "de-de", "ca-es", "cs-cz", "da-dk", "es-es", "fr-fr", "hu-hu",
+    "it-it", "nl-nl", "pl-pl", "pt-pt", "ru-ru", "sv-fi", "sv-se",
+    "uk-ua"
+)
+
+base_folder = "F:/work/skill-weather-dev"
```

Comment:
This should read from an envvar

```suggestion
base_folder = os.getenv("SKILL_DIRECTORY")
```